workshops/1.md

```diff
@@ -119,7 +119,7 @@ We need to respect the platforms, iOS, Android and web developers need to be ali
 
 We discussed that it would be great to make a Sketch plugin that would export the colors and fonts used in the app. Basically a button in Sketch that would create a Theme.swift file. One of the reasons that we want this is because in designs it's common to have colors with some transparency instead of flat colors, or backgrounds where the parent group is the one with the transparency, this is a mess to work with and having this tool would help a lot. 
 
-We also want to avoid real-time composition as much as possible, as it's harder to ensure the result is the wanted one, it's also more CPU/GPU expensive and can slow down animations.
+We also want to avoid real-time composition as much as possible, as it's harder to ensure the result is the wanted one, it's also more CPU/GPU expensive and can slow down animations. (What is real-time composition and what can designers do to stop using it?)
 
```

Comment:
Real-time composition is when we have transparent or semi-transparent views layered on top of other views. This means the system has to calculate how they blend together to be able to draw them. This is more expensive than drawing views "as is", without having to compute how it is going to look like when we blend it with the background.